### PR TITLE
feat(api,ui): organization for user

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16734.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✘

File: engine/api/group.go [hunk 1]
```diff
@@ -76,6 +78,40 @@ func (api *API) getGroupHandler() service.Handler {
 	}
 }
 
+func (api *API) getGroupExportHandler() service.Handler {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		vars := mux.Vars(r)
+
+		groupName := vars["permGroupName"]
+
+		format := FormString(r, "format")
```
Comment (fsamin) [inline engine/api/group.go:87]: just an idea: we could respect the accept header for this kind of handler

**Review by fsamin — COMMENTED** [review]
I think I expected more functional E2E tests in router_middleware_auth_permission_test.go

checkProjectPermissions, checkWorkflowPermissions, checkGroupPermissions, checkWorkerModelPermissions, etc...

other question: do services (hatcheries and workers) have organization ? do we need to track it (when we are checking runNodeJob.ExecGroups) ?

**Review by sguiheux — APPROVED** [review]

File: engine/api/group.go (hunk 1 above)
Comment (richardlt) [reply engine/api/group.go:87]: Yes could be nice to plan a change for this.

**Comment by richardlt** [comment]
> 

No changes on this middlewares, permissions still based on groups. 
The organization will only affect the ability to add people in groups or to add a group on projects, workflows or nodes.

**Review by yesnault — APPROVED** [review]

**Pull request merged**